src/lib/lib.jl

```diff
@@ -28,6 +28,14 @@ function accum(x::RefValue, y::RefValue)
   @assert x === y
   return x
 end
+function accum(x::NamedTuple, ref::RefValue)
```

Comment:
```suggestion
function accum(x::NamedTuple, ref::RefValue)
    # We do not actually do any accumulation here, because the ref will already have been mutated.
```